sdk/conn.go

```diff
@@ -1,61 +1,30 @@
 package sdk
 
 import (
-	"encoding/hex"
 	"fmt"
-	"log"
 
-	"github.com/status-im/status-go/geth/account"
-	"github.com/status-im/status-go/geth/node"
+	"github.com/valyala/gorpc"
 )
 
 // Conn : TODO ...
 type Conn struct {
-	statusNode *node.StatusNode
-	store      AccountStorer
+	rpc        *gorpc.Client
 	address    string
 	userName   string
 	channels   []*Channel
+	minimumPoW string
 }
 
-func New() *Conn {
-	return &Conn{}
-}
-
-func (c *Conn) Start(config *Config) error {
-	statusNode := node.New()
-	log.Println("Starting node...")
-
-	// Setup the storage
-	c.store = &AccountStore{}
-
-	err := statusNode.Start(config.NodeConfig)
-	if err != nil {
-		log.Fatalf("Node start failed: %v", err)
-		return err
+func New(address string) *Conn {
```

Comment:
Would name it `NewConn()`. Calling it later with `sdk.New("1.2.3.4")` makes expecting a SDK instance. Alternatively `Conn` could be renamed to `SDK`.